frappe/core/doctype/error_log/test_error_log.py

```diff
@@ -53,7 +53,6 @@ def test_ldap_exceptions(self):
 """
 
 TEST_EXCEPTIONS = {
-	"erpnext (app)": _RAW_EXC,
```

Comment:
ahh, the fix here is not to remove them but change test :laughing:

Use tuples maybe.